client/gutenberg/extensions/presets/jetpack/utils/get-jetpack-data.js

```diff
@@ -3,9 +3,23 @@
  * External Dependencies
  */
 import { get } from 'lodash';
+import apiFetch from '@wordpress/api-fetch';
 
 const JETPACK_DATA_PATH = [ 'Jetpack_Editor_Initial_State' ];
 
-export default function getJetpackData() {
-	return get( 'object' === typeof window ? window : null, JETPACK_DATA_PATH, null );
+export default async function getJetpackData() {
+	const jetpackData = get( 'object' === typeof window ? window : null, JETPACK_DATA_PATH, null );
+
+	//we may be on Gutenlypso
+	if ( ! jetpackData ) {
+		const gutenlypsoData = await apiFetch( {
+			path: '/wpcom/v2/gutenberg/available-extensions',
+			method: 'GET',
+			apiNamespace: 'wpcom/v2',
+		} );
+
+		return {
+			available_blocks: gutenlypsoData,
```

Comment:
Do we want to make the shape of the jetpackDate date to be always the same? 
on the wp-admin side we have jetpack:true, Which is suppose to help us determine if jetpack is connected. 
but I don't think it is currently being used anywhere.